ec2utils/ec2uploadimg/ec2uploadimg

```diff
@@ -366,207 +366,208 @@ try:
     ami_id = args.amiID
     running_id = args.runningID
     for region in regions:
-        setup = EC2Setup(access_key, region, secret_key, args.sessionToken,
-                         args.verbose)
-        if not ami_id and not running_id:
-            try:
-                ami_id = utils.get_from_config(args.accountName,
-                                               config,
-                                               region,
-                                               'ami',
-                                               '--ec2-ami')
-            except:
-                print('Could not determine helper AMI-ID', file=sys.stderr)
-                sys.exit(1)
-        bootkernel = args.akiID
-        if args.virtType == 'hvm':
-            bootkernel = None
-
-        if not bootkernel and args.virtType != 'hvm':
-            if args.grub2:
+        setup = None
+        try:
```

Comment:
I think the indentation is now wrong!
Also, what is the purpose of this change? I don't get it?
That you can define setup inside the loop?

Reply:
Discussed offline, fine for me